src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+if (!tree) {
+  throw new Error('List not defined');
+}
+
+for (const li of tree.querySelectorAll('li')) {
+  const span = document.createElement('span');
+  const textNode = li.firstChild;
+
+  span.append(textNode);
+  li.prepend(span);
+}
+
+tree.onclick = (e) => {
+  if (e.target.tagName !== 'SPAN') {
+    return false;
```

Comment:
Returning 'false' here is unnecessary. The function can simply return without a value, as the return value is not used.